From 66ea74afde5d07f4a9952e6fa41eb590f4a5acd0 Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 18:59:08 -0400
Subject: [fs] btrfs: remove dead comments for read_csums()

Message-id: <1407179109-13714-62-git-send-email-sandeen@redhat.com>
Patchwork-id: 87043
O-Subject: [RHEL7.1 PATCH 061/422] Btrfs: remove dead comments for read_csums()
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: Wang Shilong <wangsl.fnst@cn.fujitsu.com>

commit 663df053309c8d9200b87cc1a129729b8e97eb26
Author: Wang Shilong <wangsl.fnst@cn.fujitsu.com>
Date:   Sun Dec 15 11:39:42 2013 +0800

    Btrfs: remove dead comments for read_csums()

    Chris introduced hleper function  read_csums() and this function
    has been removed, but we forgot to remove its corresponding comments.

    Signed-off-by: Wang Shilong <wangsl.fnst@cn.fujitsu.com>
    Signed-off-by: Josef Bacik <jbacik@fb.com>
    Signed-off-by: Chris Mason <clm@fb.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/inode.c b/fs/btrfs/inode.c
index cb4ea0a..a954065 100644
--- a/fs/btrfs/inode.c
+++ b/fs/btrfs/inode.c
@@ -2543,12 +2543,6 @@ out_kfree:
  return NULL;
 }
 
-/*
- * helper function for btrfs_finish_ordered_io, this
- * just reads in some of the csum leaves to prime them into ram
- * before we start the transaction.  It limits the amount of btree
- * reads required while inside the transaction.
- */
 /* as ordered data IO finishes, this gets called so we can finish
  * an ordered extent if the range of bytes in the file it covers are
  * fully written.
-- 
1.7.1